bundles/specmate-file-connector/src/com/specmate/connectors/fileconnector/internal/FileConnector.java

```diff
@@ -161,6 +171,11 @@ public void setLogService(LogService logService) {
 
 	@Override
 	public boolean authenticate(String username, String password) {
-		return true;
+		if(user==null) {
+			return true;

```

Comment:
Shouldn't the default case, i.e. when not properly configured, be to return false?